### PR TITLE
Testing - packagingv2

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: doc/POST_INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+If encryption is enabled, your password for encryption is '__ENCRYPTION_PWD__'
```
Comment (alexAubin) [inline doc/POST_INSTALL.md:1]: Hmpf displaying user-provided password in cleartext in an UI, idk how i feel about that x_x

File: doc/POST_INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+If you want to change the frequency, have a look to the file `/etc/cron.d/__APP__`.
+
+To add recipients or to modify the files or apps to backup,please have a look to the config file `__INSTALL_DIR__/Backup_list.conf`.
+
+Please read the [documentation](https://github.com/maniackcrudelis/archivist/blob/master/Configuration.md) about the configuration of archivist for more information.
+
+You can configure this app easily by using the experimental [config-panel feature](https://__DOMAIN__/yunohost/admin/#/apps/__APP__/config-panel).  
+You can also find some specific actions for this app by using the experimental [action feature](https://__DOMAIN__/yunohost/admin/#/apps/__APP__/actions).
+
+If you're facing an issue or want to improve this app, please open a new issue in this [project](https://github.com/YunoHost-Apps/archivist_ynh).
```
Comment (alexAubin) [inline doc/POST_INSTALL.md:14]:
```suggestion
```

Actions don't exist anymore, config panel is not experimental anymore, and there should already be a link to the bugtracker in the same page this stuff is displayed

File: scripts/actions/clean_backups [hunk 3]
```diff
@@ -1,39 +1,12 @@
 #!/bin/bash
```
Comment (alexAubin) [inline scripts/actions/clean_backups:1]: This file doesn't do anything anymore

File: scripts/actions/force_backup [hunk 4]
```diff
@@ -1,45 +1,18 @@
 #!/bin/bash
```
Comment (alexAubin) [inline scripts/actions/force_backup:1]: This file doesn't do anything anymore

File: scripts/actions/reset_default_app [hunk 5]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
```
Comment (alexAubin) [inline scripts/actions/reset_default_app:1]: This file doesn't do anything anymore

File: scripts/actions/reset_default_config [hunk 6]
```diff
@@ -1,44 +1,22 @@
 #!/bin/bash
```
Comment (alexAubin) [inline scripts/actions/reset_default_config:1]: This file doesn't do anything anymore

File: doc/POST_INSTALL.md (hunk 2 above)
Comment (Salamandar) [reply doc/POST_INSTALL.md:14]: Thanks :) TBH i only copa pasta-ed the text.

**Review by alexAubin — APPROVED** [review]

**Review by alexAubin — APPROVED** [review]

**Comment by ericgaspar** [comment]
!testme

**Comment by lapineige** [comment]
:up:

**Pull request merged**